lib/Service/RoomService.php

```diff
@@ -537,8 +537,10 @@ public function verifyPassword(Room $room, string $password): array {
 		];
 	}
 
-	public function setMessageExpiration(Room $room, Participant $participant, int $seconds): void {
-		$event = new ModifyRoomEvent($room, 'messageExpiration', $seconds, null, $participant);
+	public function setMessageExpiration(Room $room, int $seconds): void {
+		$oldValue = $room->getMessageExpiration();
+		$room->setMessageExpiration($seconds);
```

Comment:
Should be done later so it's the same as in all other methods (and the before event still has the old value in the room object